src/Options.php

```diff
@@ -179,23 +179,23 @@ public function setMbDetectOrder($detectOrder): void
     }
 
     /**
-     * Gets whether the stacktrace must be auto-filled.
+     * Gets whether the stacktrace will be attached on captureMessage.
      *
      * @return bool
      */
-    public function getAutoLogStacks(): bool
+    public function getAttachStacktrace(): bool
```

Comment:
I would name this method `shouldAttachStacktrace` because it's not really a getter